### ISSUE TITLE
Food Oasis: Update alt text for Food Oasis image to adhere to WCAG

### ISSUE THREAD

**Issue by plang-psm** [hackforla/website#3999]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `Food Oasis` page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within `_projects/food-oasis.md`:

**From:**
```
Stacks of freshly harvested beets
```

**To:**
```
Food Oasis
```

**For PM, Merge Team, or Tech Lead**
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- https://www.hackforla.org/projects/food-oasis

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_projects/food-oasis.md`
- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/gh-pages/_projects/food-oasis.md
- This issue is part of epic #3866

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by Alsuang** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"

-Just submitted a pull request. I finished changing the image alt property value to 'Food Oasis'.

Blockers: "Difficulties or errors encountered."
-None encountered.

Availability: "How much time will you have this week to work on this issue?"
-N/A. Already finished issue.

ETA: "When do you expect this issue to be completed?"
-Took around an 1 hour. Most of the time reading documentation for Git .
Pictures (optional): "Add any pictures of the visual changes made to the site so far."

**Comment by Alsuang** [comment]
Availability for this week: Mon-Sun 5-11pm
My estimated ETA for completing this issue: 1hr

**Comment by Alsuang** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

It took around 4 hours to complete these tasks, which also includes the onboarding meeting.
